var/spack/repos/builtin/packages/hpx/package.py

```diff
@@ -266,6 +266,12 @@ def cmake_args(self):
         # Instrumentation
         args += self.instrumentation_args()
 
+        if "instrumentation=thread_debug" in spec:
+            args += [
+                self.define("HPX_WITH_THREAD+DEBUG_INFO", True),
```

Comment:
This looks like a typo:
```suggestion
                self.define("HPX_WITH_THREAD_DEBUG_INFO", True),
```